File: lists/selection_sort.js
let A = [64, 25, 12, 22, 11];

const sorter = arr => {
    for(let i in arr) {
        let min_idx = i;

        for (let j = i + 1; j < arr.length; j++) {
            if (arr[min_idx] > arr[j]) {
                min_idx = j;
            }
        }
        [arr[i], arr[min_idx]] = [arr[min_idx], arr[i]];
    }
    for (let i in arr) {
        console.log(arr[i]);
    }
}


sorter(A);

File: lists/selection_sort.py
import sys


A = [64, 25, 12, 22, 11]

def sorter(l):
    for i in range(len(l)):
        min_idx = 1

        for j in range(i + 1, len(l)):
            if l[min_idx] > l[j]:
                min_idx = j 

        l[i], l[min_idx] = l[min_idx], l[i]

    for i in range(len(l)):
        print(l[i])

print(sorter(A))